src/main/java/org/xstefank/control/RepositoryFormatControl.java

```diff
@@ -0,0 +1,18 @@
+package org.xstefank.control;
+
+import org.xstefank.model.yaml.FormatConfig;
+
+public class RepositoryFormatControl implements Control {
+
+    private final String REGEX = "^[A-z,0-9].*/[A-z,0-9,-].*$";
+    private final String MESSAGE = "Bad repository format in configuration file";
+
+    public String control(FormatConfig formatConfig) {
+        String repoFormat = formatConfig.getRepository();
+        if (repoFormat.matches(REGEX) && repoFormat.split("/").length == 2) {
+            return null;
```

Comment:
returning null is generally a bad practice, especially if it is undocumented, the caller of this method can easily get NPE